src/main/java/net/sf/jabref/gui/FXAlert.java

```diff
@@ -71,26 +72,29 @@ public void windowGainedFocus(WindowEvent e) {
         }
     };
 
-
-    public FXAlert(AlertType type, String title, Image image) {
```

Comment:
I would leave all the old constructors and call the new overloads with `isModal=true` since this looks like the default behavior. For example:
`public FXAlert(AlertType type, String title) { this(type, title, true); }`.